atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTableResponse.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.sweep;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.palantir.atlasdb.keyvalue.api.SweepResults;
+
+@Value.Immutable
+@JsonSerialize(as=ImmutableSweepTableResponse.class)
+@JsonDeserialize(as=ImmutableSweepTableResponse.class)
+public interface SweepTableResponse {
+
+    //Optional<String> nextStartRow();
```

Comment:
`DropwizardClientRule` does not use an object mapper that supports optionals so this makes all the tests fails... need to fix this